dv/uvm/sim.py

```diff
@@ -178,8 +178,8 @@ def compare(test_list, iss, output_dir, verbose):
       logging.info("Comparing %s/DUT sim result : %s" % (iss, elf))
       run_cmd(("echo 'Test binary: %s' >> %s" % (elf, report)))
       uvm_log = ("%s/rtl_sim/%s.%d/sim.log" % (output_dir, test['test'], i))
-      rtl_log = ("%s/rtl_sim/%s.%d/trace_core_00000000.log" % (output_dir, test['test'], i))
-      rtl_csv = ("%s/rtl_sim/%s.%d/trace_core_00000000.csv" % (output_dir, test['test'], i))
+      rtl_log = ("%s/rtl_sim/%s.%d/trace_core__00000000.log" % (output_dir, test['test'], i))
+      rtl_csv = ("%s/rtl_sim/%s.%d/trace_core__00000000.csv" % (output_dir, test['test'], i))
```

Comment:
Oh, that double `__` looks unintentional. I'll double-check.

Reply:
Opened https://github.com/lowRISC/ibex/pull/367 to get this fixed.

Reply:
Ah sure, I've approved your related filename PR and reverted the filename in this script.